### DISCUSSION TITLE
Release cadence for Faster.Core nuget

### DISCUSSION THREAD

**Discussion by vangarp** [microsoft/FASTER#643] [General]
Hey folks, thanks for the awesome work you are doing on this project. Love to see that you are actively addressing any issues and questions reported within just a few days. It gives us a lot of confidence in taking dependency on faster. Do we have a cadence on releases of the Faster.Core nuget package? A few fixes I am tracking and am looking for are in, but wondering when we might have the next release which will include those fixes.

**Comment by badrishc** [comment]
Our release cadence for 1.9.x is on an as needed basis. We plan to trigger one today to cover the recent check-ins. We are currently focusing on our next major release preview (2.0) at https://github.com/microsoft/FASTER/pull/563, which has several new features.

**Comment by badrishc** [comment]
Update: 1.9.11 should now be available.

**Reply by vangarp** [reply]
thanks!
